### PR TITLE
Lesson 3 - Task1, Task2, Task6 -  commit

### PR DESCRIPTION
**Pull request by ButrynVasiliy**

### CODE REVIEW

**Comment by ButrynVasiliy** [comment]
Task1, Task2, Task6,

File: src/Task1Lesson3.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        String s = example.substring(0, 2);
+        System.out.println(s);
+        String a = "A";
+        String c = "";
+        // String c for isEmpty
+        System.out.println(c.isEmpty());
+
+        if (a.length() <= 2){
+            System.out.println("String " +a);
+        }
```
Comment (Evangard) [inline src/Task1Lesson3.java]: Your code should look like:

    public static void main(String[] args) {
        System.out.println(getStr("AE_1381"));
        System.out.println(getStr("A"));
        System.out.println(getStr(""));
    }


    public static String getStr(String str) {

    }
    
    
    In method getStr should be a logic.

File: src/Task4Lesson3.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+public class Task4Lesson3 {
+    //4. Given a string "ABCDEFGHIJK".
+    // You need to make this string lowercased and remove all vowels.
+    public static void main(String[] args){
+        String alfabet = "ABCDEFGHIJK";
+        String newAlfabet = new String(alfabet.toLowerCase());
+        System.out.println("toLowerCase = " +newAlfabet);
```
Comment (Evangard) [inline src/Task4Lesson3.java]: No need to create newAlfabet

File: src/Task1Lesson3.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+public class Task1Lesson3 {
```
Comment (Evangard) [inline src/Task1Lesson3.java:1]: Why is this class commented?

File: src/Task5Lesson3.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+        String a = "hop";
+        String b = "hello";
+                if ( a.length() < b.length()) {
+                    StringBuilder result = new StringBuilder(a + " " + b + " " + a);
+                    System.out.println(result);
+                }
+                else {
+                    StringBuilder resultOther = new StringBuilder(b + " " + a + " " + b);
+                    System.out.println(resultOther);
```
Comment (Evangard) [inline src/Task5Lesson3.java]: You duplicate code in lines 11,12 and 15, 16. Create result variable before if/else and use println after if/else.

File: src/Task6Lesson3.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    //   login = "John Smith"
+    //   domain = "example.com"
+    public static void main(String[] args){
+        String email = "john_smith@example.com";
+        int index1 = email.indexOf("_");
+        int index = email.indexOf("@");
+        String loginName = email.substring(0, index1);
+        String loginSername = email.substring(index1+1, index);
+        String domain = email.substring(index+1, email.length());
+        System.out.println("\'login\' - " +loginName.replace("j", "J") + " " +loginSername.replace("s", "S"));
```
Comment (Evangard) [inline src/Task6Lesson3.java]: Better to use substring and uppercase instead of replace.

File: src/Task7Lesson3.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    //   Example:
+    //   Den
+    //   John
+    //   Will
+    //   Kate
+    //   ...
+    public static void main(String[] args){
+
+        String name = "Den, John, Will, Kate, Adam, Robin";
+        String all = String.join("\n","Den", "John", "Will", "Kate", "Adam", "Robin");
```
Comment (Evangard) [inline src/Task7Lesson3.java]: This one is good. Try one more way (split).

File: src/Task8.java [hunk 7]
```diff
@@ -2,9 +2,9 @@ public class Task8 {
     public static void main(String[] args){
 
         for (int i = 0; i <= 100; i++){
-            if (i % 3 == 0 && i % 5 == 0 && i % 2 != 0 )
+            if (i % 3 == 0 && i % 2 != 0 || i % 5 == 0 && i % 2 != 0 )
```
Comment (Evangard) [inline src/Task8.java]: Can be shorter 
if ((i % 3 == 0 || i % 5 == 0) && i % 2 != 0)

**Pull request merged**